src/components/stepFour/utils.js

```diff
@@ -1008,6 +1008,65 @@ export const SUMMARY_FILE_CONTENTS = networkID => {
     return []
   }
 
+  const isDutchStrategy = crowdsaleStore.strategy === CROWDSALE_STRATEGIES.DUTCH_AUCTION
+  const labelIdx = isDutchStrategy ? 'DutchIdx' : 'MintedCappedIdx'
+  const labelCrowdsale = isDutchStrategy ? 'Dutch Crowdsale' : 'Sale'
+  const labelToken = isDutchStrategy ? 'Dutch Token' : 'Token'
+  const labelSaleManager = isDutchStrategy ? 'Dutch Crowdsale manager' : 'Sale manager'
```

Comment:
As Dutch strategy has no managers smart-contracts, here and in the next line, we could remove conditional operator and just leave values for Minted capped strategy

Reply:
Change done